templates/web/merton/footer_extra_js.html

```diff
@@ -0,0 +1,4 @@
+[% PROCESS 'footer_extra_js_base.html' cobrand_js=1 validation=1 tfl=1 roadworks=1 %]
+[% scripts.push(
```

Comment:
The footer_extra_js_base.html includes /cobrands/merton/assets.js because of the `cobrand_js=1` parameter, so this addition of the same JS file is unnecessary.